build/FrameworkPackage/CustomizedPriConfig/pri.resfiles

```diff
@@ -0,0 +1 @@
+Microsoft.UI.Xaml\DensityStyles\Compact.xaml
```

Comment:
This should be the .xbf file, I think.

Reply:
I see that we aren't currently passing this file to the XBF compiler anywhere. I don't know if it's easy to do this outside of the MSBuild environment.

---
In reply to: [277879139](https://github.com/Microsoft/microsoft-ui-xaml/pull/608#discussion_r277879139) [](ancestors = 277879139)